storage/crdb/log_storage.go

```diff
@@ -0,0 +1,848 @@
+// Copyright 2022 <TBD>
```

Comment:
I think these lines should retain the the date from the original mysql files they derive from, but could be "Copyright XXXX The Trillian Authors" (and you can add an entry in the AUTHORS/CONTRIBUTORS files).